data/transportation/transit/index.html

```diff
@@ -27,44 +27,39 @@
     - url: https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mMF9ZUkdiMEpsb3M&export=download
       label: "Bus Stops: Shapefile"
   updates:
-    - "Updates to this dataset are published periodically.<br>
-      Last Update: January 2017"
+    - "January, 2017"
 UTACommuterRailRoutes:
   downloads:
     - url: https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mUWczdHRhRkRSbVk&export=download
       label: "Commuter Rail Routes: File Geodatabase"
     - url: https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mOTdYaXkxMWNtYzA&export=download
       label: "Commuter Rail Routes: Shapefile"
   updates:
-    - "Updates to this dataset are published periodically.<br>
-      Last Update: May, 2013"
+    - "May, 2013"
 UTACommuterRailStations:
   downloads:
     - url: https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mYzUtd3BFcE5LN3M&export=download
       label: "Commuter Rail Stations: File Geodatabase"
     - url: https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mSmhydGg1NjVBSXM&export=download
       label: "Commuter Rail Stations: Shapefile"
   updates:
-    - "Updates to this dataset are published periodically.<br>
-      Last Update: May, 2013"
+    - "May, 2013"
```

Comment:
FYI these only need to be quoted if they use colons and some other yaml things. I've started to make a habit of quoting everything.